src/app_service/service/privacy/service.nim

```diff
@@ -130,3 +130,24 @@ QtObject:
       return
 
     return mnemonics[index]
+
+  proc validatePassword*(self: Service, password: string): bool =
+    try:
+      var isValid = false
```

Comment:
@Khushboo-dev-cpp please remove this `isValid` variable (it's cleaner that way) and in each `then` block of your conditions below just to `return false` and at the end of the `try` block `return true`. 

Also as a final step either in `except` section or as the last line of this method do `return false` (that's in case an error has been thrown).

Reply:
done @saledjenic thanks for pointing it out :)